dev-python/spyder-notebook/spyder-notebook-0.2.1.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Jupyter notebook integration with Spyder "
+HOMEPAGE="https://github.com/spyder-ide/spyder-notebook"
+SRC_URI="https://github.com/spyder-ide/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+IUSE="doc"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-python/nbformat[${PYTHON_USEDEP}]
+	dev-python/notebook[${PYTHON_USEDEP}]
+	dev-python/psutil[${PYTHON_USEDEP}]
+	dev-python/QtPy[webkit,${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	>=dev-python/spyder-4.0.0[${PYTHON_USEDEP}]"
+
+# Tests do not work inside virtx/emerge for some reason, core dumped
```

Comment:
Define the phase, then add `RESTRICT="test"` with this comment on top of it. It's easier for others to test it when it's defined in the ebuild, but just restricted.